packages/ssr-web/app/templates/pay.hbs

```diff
@@ -1,23 +1,29 @@
 {{#let
-  (concat (or this.merchantInfo.name @model.merchantSafe.address) ' requests payment')
-  (concat 'Pay ' this.cleanedAmounts.displayed.amount)
-  as |title description|
+  (or this.merchantInfo.name @model.merchantSafe.address)
+  as |business|
 }}
-  {{page-title title replace=true}}
+  {{#let
+    (concat business ' requests payment')
+    (concat 'Use Card Wallet to pay ' (or this.cleanedAmounts.displayed.amount business))
```

Comment:
Maybe we should get input on this? I had “Card Wallet” in a previous iteration and was [asked](https://discord.com/channels/584043165066199050/813819872177815572/945823567864545282) to remove it 😆

Reply:
@Aierie please add a screenshot to this PR and ask Kiel to provide input on the copy.